hostfactory/host_provider/src/cyclecloud_provider.py

```diff
@@ -794,34 +801,58 @@ def _deperecated_terminate_status(self, input_json):
         # executing is a terminal state.
         
         response = {"requests": []}
+        request_status = RequestStates.complete
         # needs to be a [] when we return
         with self.terminate_json as terminate_requests:
             
             self._cleanup_expired_requests(terminate_requests, self.termination_timeout, "terminated")
             
             termination_ids = [r["requestId"] for r in input_json["requests"] if r["requestId"]]
             machines_to_terminate = []
-            try:
-                for termination_id in termination_ids:
-                    if termination_id in terminate_requests:
-                        termination = terminate_requests[termination_id]
-                        if not termination.get("terminated"):
-                            for machine_id, name in termination["machines"].items():
-                                machines_to_terminate.append({"machineId": machine_id, "name": name})
-                
-                if machines_to_terminate:
-                    logger.warning("Re-attempting termination of nodes %s", machines_to_terminate)
-                    self.cluster.shutdown_nodes(machines_to_terminate)
-                    
-                for termination_id in termination_ids:
-                    if termination_id in terminate_requests:
-                        termination = terminate_requests[termination_id]
-                        termination["terminated"] = True
-                        
-            except Exception:
-                request_status = RequestStates.running
-                logger.exception("Could not terminate nodes with ids %s. Will retry", machines_to_terminate)
+            for termination_id in termination_ids:
+                if termination_id in terminate_requests:
+                    termination = terminate_requests[termination_id]
+                    if not termination.get("terminated"):
+                        for machine_id, name in termination["machines"].items():
+                            machines_to_terminate.append({"machineId": machine_id, "name": name})
             
+            if machines_to_terminate:
+                logger.warning("Re-attempting termination of nodes %s", machines_to_terminate)
+                try:
+                    self.cluster.terminate(machines_to_terminate)
```

Comment:
Why did we change from shutdown _nodes to terminate? Do we not support ShutdownMethod=Deallocation?

Reply:
we do support shutdown, I will update that. Actually this was directly merged so I forgot to update. I will update it to shutdown. Thanks